common/scala/src/main/scala/whisk/common/TransactionId.scala

```diff
@@ -128,13 +138,13 @@ protected case class TransactionMetadata(val id: Long, val start: Instant)
 
 object TransactionId {
     val unknown = TransactionId(0)
-    val testing = TransactionId(-1)         // Common id for for unit testing
```

Comment:
i dont know how to get the IDE to preserve the comment alignment here - would be nice to restore.